src/OpenCV_Mat.jl

```diff
@@ -194,18 +194,25 @@ function pixset(img, row::Int, col::Int, value)
     @cxx at_d(img, row, col, value);
   # RGB images (value::  std::vector<double>)
   elseif (cvtypeval(img) == CV_8SC3)
+    cppvec = tostdvec(value)
```

Comment:
Where is `cppvec` passed next?   I see that you pass `colorvec` to `pixset` and then convert the julia array to an `std::vector`with `tostdvec`.  It seems then you should be passing it further e.g, `@cxx at_vc(img, row, col, cppvec);` as so on to the other cases.